components/NewReplySection/Desktop.js

```diff
@@ -105,13 +105,21 @@ const Desktop = ({
       )}
       {selectedTab === 1 && (
         <>
+          <Box
+            display="flex"
+            alignItems="center"
+            justifyContent="space-between"
```

Comment:
[Trivial] Since all its child have `flex-grow`, we don't need `justifyContent` here